.travis.yml

```diff
@@ -1,6 +1,7 @@
 install:
  - mvn clean; java -version
 script:
+ - mvn -B -Dmaven.javadoc.skip=false javadoc:javadoc
```

Comment:
I guess this should go after the build is done, right? So we should switch the lines. Also do we need to execute it with every WF in matrix combinations? Perhaps something like https://docs.travis-ci.com/user/build-stages/matrix-expansion/ ?

Reply:
Made it as another stage. The first run is test in the test stage, the second run is javadoc in javadoc stage. Unfortunately I have to fix javadoc for Java 11.